src/test/java/edu/cornell/kfs/module/purap/iwant/xml/fixture/IWantItemFixture.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.module.purap.iwant.xml.fixture;
+
+import org.kuali.kfs.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.module.purap.iwant.xml.IWantItemXml;
+
+public enum IWantItemFixture {
+
+    ITEM_TEST("unit of measure", "cat number", "item description", new KualiDecimal(5.29), "code",
```

Comment:
In my prior review, what I meant was that you could just pass in a `double` value at this spot, and then let the constructor handle converting it to the desired `BigDecimal`/`KualiDecimal` type. (The same goes for the spot in the transaction line fixture.) Sorry if that wasn't clear.

If you'd rather keep it as-is so that the `KualiDecimal` setup remains at this spot, please let me know.

Reply:
lets leave this as is for now

Reply:
Ok, I'm fine with keeping this part as-is.